spec/e2e/nodesets/ubuntu-22.04.yml

```diff
@@ -0,0 +1,36 @@
+HOSTS:
+  ubuntu2204:
+    roles:
+      - agent
+    platform: ubuntu-22.04-amd64
+    hypervisor : docker
+    image: ubuntu:22.04
+    docker_preserve_image: true
+    docker_cmd: '["/sbin/init"]'
+    docker_image_commands:
+      - "rm -f /etc/dpkg/dpkg.cfg.d/excludes"
+      - 'apt-get install -y wget net-tools locales apt-transport-https ca-certificates'
+      - 'locale-gen en_US.UTF-8'
+    docker_env:
+      - LANG=en_US.UTF-8
+      - LANGUAGE=en_US.UTF-8
+      - LC_ALL=en_US.UTF-8
+      - DEBIAN_FRONTEND=noninteractive
```

Comment:
22 seems to be hanging on this prompt below. I feel like this env variable isn't working in 22.

```
  debconf: (TERM is not set, so the dialog frontend is not usable.)
  debconf: falling back to frontend: Readline
  Configuring tzdata
  ------------------
  Please select the geographic area in which you live. Subsequent configuration  
  questions will narrow this down by presenting a list of cities, representing  
  the time zones in which they are located.
  
    1. Africa   3. Antarctica  5. Arctic  7. Atlantic  9. Indian    11. US
    2. America  4. Australia   6. Asia    8. Europe    10. Pacific  12. Etc
```